### DISCUSSION TITLE
How to run e2e test in my local environment

### DISCUSSION THREAD

**Discussion by duanmengkk** [clusterpedia-io/clusterpedia#279] [Q&A] [answered]
How to run e2e test in my local environment ,I did not see any test case code in clusterpedia?  @Iceber

**Comment by Iceber** [comment]
@wzshiming PTAL :heart:

**Comment by wzshiming** [comment, marked as answer]
Hi

There is currently a basic test case in https://github.com/clusterpedia-io/clusterpedia/blob/main/test/cases/workable.test.sh


Just run this script and it will run all e2e cases
https://github.com/clusterpedia-io/clusterpedia/blob/main/test/test.sh
